cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -11,12 +11,129 @@ Docker Image
 
 
 Docker is one of the easiest ways to start working with CDAP without having to manually
-configure anything. A Docker image with the CDAP SDK pre-installed is available on the Docker Hub
+configure anything. A Docker container with the CDAP SDK pre-installed is available on the Docker Hub
```

Comment:
This was correct before.

The thing that we create and ship to Docker Hub is an image. That image is taken and instantiated as a container. So... running == container, the thing you build/pull is an image.